fMRIVolume/scripts/MotionCorrection.sh

```diff
@@ -44,6 +59,25 @@ case "$MotionCorrectionType" in
     ;;
 esac
 
+
+# Run nonlinear registration if needed
```

Comment:
Let's provide more context here, to help users trying to follow the code:
```
# If registering across runs, perform nonlinear registration if requested.
# (If using linear registration, don't need to do anything extra here, since
# linear registration is handled implicitly via the motion correction).
```

Reply:
Added.